doc/en/user/source/services/generic_vendor_options.txt

```diff
@@ -0,0 +1,29 @@
+General Vendor Options
+----------------------
+
+These vendor options are available for all operations.
+
+content-disposition
+^^^^^^^^^^^^^^^^^^^
+
+The ``content-disposition`` parameter directs how a web browser directed to handle returned content. The syntax is::
+
+   content-dispoition=<dispoition>
+
+Where ``content-dispoition=attachment`` to direct the browser to save the content to disk (even if it is a format such as json or text which it may be wlling to display directly).
```

Comment:
```suggestion
Where ``content-disposition=attachment`` to direct the browser to save the content to disk (even if it is a format such as JSON or plain text which it may be willing to display directly).
```

Reply:
I will shorten the sentence to just "save to disk".